### DISCUSSION TITLE
SSL certification problem

### DISCUSSION THREAD

**Discussion by Emmanueldlc24** [community/community#75637] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I am new to the programming world, I am needing guidance in understanding/resolving the SSL certification problem: unable to get local issuer certification

**Comment by Imen-Frigui** [comment]
here is some guides that could help
if you could give more detail all be happy to help
https://komodor.com/learn/how-to-fix-ssl-certificate-problem-unable-to-get-local-issuer-certificate-git-error/
https://stackoverflow.com/questions/23885449/unable-to-resolve-unable-to-get-local-issuer-certificate-using-git-on-windows

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @Emmanueldlc24! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
